go.mod

```diff
@@ -4,22 +4,23 @@ go 1.13
 
 require (
 	github.com/blang/semver v3.5.0+incompatible
-	github.com/evanphx/json-patch v4.5.0+incompatible
+	github.com/evanphx/json-patch v4.9.0+incompatible
 	github.com/go-git/go-git/v5 v5.1.0
+	github.com/go-logr/logr v0.2.1
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/tools v0.0.0-20200714190737-9048b464a08d
 	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
-	k8s.io/api v0.18.4
-	k8s.io/apimachinery v0.18.4
-	k8s.io/cli-runtime v0.18.4
-	k8s.io/client-go v0.18.4
+	k8s.io/api v0.19.2
+	k8s.io/apimachinery v0.19.2
+	k8s.io/cli-runtime v0.19.2
+	k8s.io/client-go v0.19.2
 	k8s.io/klog v1.0.0
-	k8s.io/kubectl v0.18.4
+	k8s.io/kubectl v0.19.2
 	sigs.k8s.io/cli-utils v0.16.0
-	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/controller-runtime v0.7.0-alpha.4
```

Comment:
@estroz IMO we need wait for the stable release to be able to bump the dep.

Reply:
Is there a good reason to wait? I don't expect any changes to go in aside from bugfixes at this point, although it is technically an alpha release. Unless there is a plan to release this project soon, bumping to alpha now then bumping to stable before a release makes sense to me since the alpha bump will only ever be on master. This also helps test out the controller-runtime v0.7.0 release.

Reply:
We discussed that. I think it is fine we move forward. 
Really thank you for your collab.